bedrock/firefox/views.py

```diff
@@ -636,12 +636,15 @@ def download_thanks(request):
     locale = l10n_utils.get_locale(request)
     variant = request.GET.get('v', None)
     show_newsletter = locale in ['en-US', 'en-GB', 'en-CA', 'en-ZA', 'es-ES', 'es-AR', 'es-CL', 'es-MX', 'pt-BR', 'fr', 'ru', 'id', 'de', 'pl']
-    pre_download_locales = ['en-US', 'en-GB', 'en-CA', 'en-ZA', 'de', 'fr']
+    pre_download_locales = ['fr']
```

Comment:
What's going on here? Rebase weirdness?

Reply:
Which is to say, did you intend to change the value of the `pre_download_locales` in this PR, I didn't think the experiment is related to this experiment. Maybe these lines have gotten out of sync through a rebase?

Reply:
Sorry, I mean to leave a comment for the reviewer here and totally forgot. I'm cleaning up some old experiment code here to make way for this new test to run (with the exception of `fr`, all pre-download experiments have ended).

Reply:
👍